### PR TITLE
updated overview page for payables

### PR DESCRIPTION
**Pull request by jmyorston**
# Description

updated the payables page - changed some of the text and added feature cards.
`/usecases/summary/automating-payables`

## Type of change

Please delete options that are not relevant.

- [x] New document(s)/updating existing
- [ ] Fixes
- [ ] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: docs/usecases/summary/automating-payables.mdx [hunk 1]
```diff
@@ -1,36 +1,77 @@
 ---
 title: "Automating payables with Codat"
 sidebar_label: "Overview"
-description: "An overview of the payables automation use case supported by Codat"
+description: "Streamline your customers' accounts payable workflow."
```
Comment (pzaichkina) [inline docs/usecases/summary/automating-payables.mdx:4]:
```suggestion
description: "Streamline your customers' accounts payable workflow"

```

File: docs/usecases/summary/automating-payables.mdx [hunk 2]
```diff
@@ -1,36 +1,77 @@
...
 title: "Automating payables with Codat"
 sidebar_label: "Overview"
-description: "An overview of the payables automation use case supported by Codat"
+description: "Streamline your customers' accounts payable workflow."
 ---
 
 import Products from "@components/global/Products";
 
-Engage Codat and automate your customers' accounts payable processes by reflecting outstanding and paid bills and invoices from business software spend and suppliers in their accounting platforms.
+By integrating your customers accounting software, supplier information and invoices and payments can be easily kept in sync.
```
Comment (pzaichkina) [inline docs/usecases/summary/automating-payables.mdx:9]:
```suggestion
By integrating your customers' accounting software, their supplier information, invoices, and payments can be easily kept in sync. 

```

File: docs/usecases/summary/automating-payables.mdx [hunk 3]
```diff
@@ -1,36 +1,77 @@
...
-description: "An overview of the payables automation use case supported by Codat"
+description: "Streamline your customers' accounts payable workflow."
 ---
 
 import Products from "@components/global/Products";
 
-Engage Codat and automate your customers' accounts payable processes by reflecting outstanding and paid bills and invoices from business software spend and suppliers in their accounting platforms.
+By integrating your customers accounting software, supplier information and invoices and payments can be easily kept in sync. 
+
+This automation replaces manual tasks and eliminates duplication of effort, making it easier for your customers and freeing up time for them to concentrate on running their business.
```
Comment (pzaichkina) [inline docs/usecases/summary/automating-payables.mdx:11]: making what easier for the customers? this part of the phrase does not currently show any value. 
I think it's sufficient to just say 
"This automation replaces manual tasks and eliminates duplication of effort,  freeing up your customers' time so they can concentrate on running their business."

File: docs/usecases/summary/automating-payables.mdx [hunk 4]
```diff
@@ -1,36 +1,77 @@
...
+  </li>
 
-We have done the heavy lifting for you by building integrations with a standardized data model to the accounting platforms your customers already use. This gives you access to real-time data that you can pull, create, or update to support your use case.
+  <li className="card">
+    <div class="header">
+      <img
+        src="/img/wp-icons/Files.png"
+        className="mini-icon"
+      />
+      <h3>Create Bills</h3>
```
Comment (pzaichkina) [inline docs/usecases/summary/automating-payables.mdx:44]:
```suggestion
      <h3>Create bills</h3>

```

File: docs/usecases/summary/automating-payables.mdx [hunk 5]
```diff
@@ -1,36 +1,77 @@
...
+    </p>
+  </li>
+
+  <li className="card">
+    <div class="header">
+      <img
+        src="/img/wp-icons/Graph-1.png"
+        className="mini-icon"
+      />
+      <h3>Removing Repetition</h3>
```
Comment (pzaichkina) [inline docs/usecases/summary/automating-payables.mdx:57]:
```suggestion
      <h3>Removing repetition</h3>

```

File: docs/usecases/summary/automating-payables.mdx [hunk 6]
```diff
@@ -1,36 +1,77 @@
...
 ### How Codat can help
 
-We can support B2B payment and invoicing software companies, payment providers, and vertical SaaS companies in their activity by leveraging our [Accounting API](/accounting-api/overview) product. 
+Codat supports B2B payment and invoicing companies, payment providers, and vertical SaaS businesses with their integration requirements by leveraging the [Accounting API](/accounting-api/overview) product. 
 
-It enables businesses to track and manage their spend, typically including the whole purchasing flow from the creation of the PO through to a bill payment.
+It enables small and medium sized businesses to track and manage their spend with their chosen application and ensure that their financial reports are kept up to date.
 
-#### Relevant products
+#### Supported Features
```
Comment (pzaichkina) [inline docs/usecases/summary/automating-payables.mdx:19]:
```suggestion
#### Supported features

```

File: docs/usecases/summary/automating-payables.mdx [hunk 7]
```diff
@@ -1,36 +1,77 @@
...
+  <li className="card">
+    <div class="header">
+      <img
+        src="/img/wp-icons/Files.png"
+        className="mini-icon"
+      />
+      <h3>Create Bills</h3>
+    </div>
+    <p>
+      Enable a vendor bill to be created within their application and then synchronized to customers’ accounting platforms.
```
Comment (pzaichkina) [inline docs/usecases/summary/automating-payables.mdx:47]:
```suggestion
      Enable your customers to create a vendor bill within their application, and synchronize it to their accounting platform.
```

File: docs/usecases/summary/automating-payables.mdx [hunk 8]
```diff
@@ -1,36 +1,77 @@
...
+  <li className="card">
+    <div class="header">
+      <img
+        src="/img/wp-icons/Graph-1.png"
+        className="mini-icon"
+      />
+      <h3>Removing Repetition</h3>
+    </div>
+    <p>
+      Remove repetition by reconciling the payments against supplier invoices in your customers accoutning software.
```
Comment (pzaichkina) [inline docs/usecases/summary/automating-payables.mdx:60]:
```suggestion
      Reconcile payments against supplier invoices in your customers' accounting software, reducing the chance of duplication.
```

**Review by pzaichkina — CHANGES_REQUESTED** [review]

File: docs/usecases/summary/automating-payables.mdx (hunk 3 above)
Comment (jmyorston) [reply docs/usecases/summary/automating-payables.mdx]: @pzaichkina great idea - updated

File: docs/usecases/summary/automating-payables.mdx (hunk 2 above)
Comment (d-laing) [reply docs/usecases/summary/automating-payables.mdx]: Alternative phrasing:

Integrate with your customers' accounting software to easily keep their supplier information, invoices, and payments in sync.

**Review by pzaichkina — APPROVED** [review]

**Pull request merged**